File: Haskell/beginner/1015.hs
-- Importing printf
import Text.Printf

-- Main Function
main = do
    -- Requesting values and assign into variables
    first_line <- getLine
    second_line <- getLine

    -- Extracting values
    let [x1, y1] = map read (words first_line) :: [Double]
    let [x2, y2] = map read (words second_line) :: [Double]

    -- Finding distance
    let distance = sqrt ((x2 - x1) ^ 2 + (y2 - y1) ^ 2)

    -- Showing results
    printf "%.4f\n" distance

-- By Fernando Serra
-- https://github.com/fernandosserra

File: C/beginner/1015.c
/*Including Libs*/
#include <stdio.h>
#include <math.h>

/*Main Function*/
int main (){
    /*Variable declaration and initialization*/
    double x1, y1, x2, y2, distance;
    
    /*Requesting values to user*/
    scanf("%lf %lf" , &x1, &y1);
    scanf("%lf %lf" , &x2, &y2);

    /*Finding distance*/
    distance = ((x2 - x1) * (x2 - x1)) + ((y2 - y1) * (y2 - y1));
    distance = sqrt(distance);
    
    /*Printing distance*/
    printf("%.4lf\n", distance);

    /*Ending*/
    return 0;
}

/*By Fernando Serra*/
/*https://github.com/fernandosserra*/